src/wp-includes/class-wp-term.php

```diff
@@ -99,6 +99,22 @@ final class WP_Term {
 	 */
 	public $filter = 'raw';
 
+	/**
+	 * Stores the sanitized term data.
+	 *
+	 * @since 6.6.0
+	 * @var stdClass|null
+	 */
+	private $data;
```

Comment:
Are there any arguments for making the `$data` property protected instead of private?
The `WP_Term` class is declared as final, so it is unlikely that it will be extended.